packages/supi/src/install/index.ts

```diff
@@ -837,9 +837,17 @@ async function installInContext (
       })
     }
 
-    if (!opts.lockfileOnly) {
-      await Promise.all(projectsToLink.map(linkBinsOfImporter))
-    }
+    await Promise.all(projectsToLink.map(async (project, index) => {
+      const projectToInstall = projects[index]
+      if (opts.global && projectToInstall.mutation === 'installSome') {
+        const linkedPackegs = await linkBinsOfImporter(project)
+        projectToInstall.dependencySelectors.forEach(pkg => {
+          if (!linkedPackegs?.includes(pkg)) {
+            logger.warn({ message: `The globally installed package ${pkg} has no bins.`, prefix: opts.lockfileDir })
```

Comment:
Let's use a similar message to Yarn's
```suggestion
            logger.warn({ message: `${pkg} has no binaries`, prefix: opts.lockfileDir })
```